src/components/Card/Card.story.jsx

```diff
@@ -100,7 +100,7 @@ storiesOf('Watson IoT|Card', module)
           size={size}
           isLoading={boolean('isLoading', false)}
           isEmpty={boolean('isEmpty', false)}
-          isEditable={boolean('isEditable', true)}
+          isEditable={boolean('isEditable', false)}
           isExpanded={boolean('isExpanded', false)}
```

Comment:
I know you didn't change the `isExpanded` knob here, but it seems backwards in storybook? When isExpanded is checked, the card is not expanded visually.

| - | - |
| - | - |
| ![image](https://user-images.githubusercontent.com/3360588/70270643-77141b00-176a-11ea-8a8a-1d723f0413d0.png) | ![image](https://user-images.githubusercontent.com/3360588/70270666-872bfa80-176a-11ea-85b2-93b27ca6f55c.png) |

Reply:
can I fix that in my next track?  I probably shouldn't have fixed the other bug since this is supposed to just be a plain merge...